routers/Arista.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * Looking Glass - An easy to deploy Looking Glass
+ * Copyright (C) 2014-2019 Guillaume Mazoyer <gmazoyer@gravitons.in>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+ */
+
+require_once 'router.php';
+require_once 'includes/command_builder.php';
+require_once 'includes/utils.php';
+
+final class Arista extends Router
+{
+    protected function build_bgp($parameter): array
+    {
+        $cmd = new CommandBuilder();
+        $cmd->add('show');
+
+        if (match_ipv6($parameter, false)) {
+            $cmd->add('ipv6 bgp', $parameter);
+        }
+        if (match_ipv4($parameter, false)) {
+            $cmd->add('ip bgp', $parameter);
+        }
+        if ($this->config['bgp_detail']) {
+            $cmd->add('detail');
+        }
+
+        return array($cmd);
+    }
+
+    protected function build_aspath_regexp($parameter): array
+    {
+        #$parameter = quote($parameter);
+        $cmd = new CommandBuilder();
+        $cmd->add('show');
+        $cmd->add('ip bgp regex', $parameter);
+
+        return array($cmd);
+    }
+
+    protected function build_aspath_regexp6($parameter): array
```

Comment:
Consider adding the IPv6 specific AS path regexp command to the existing function. You can take a look at the [`juniper.php`](https://github.com/respawner/looking-glass/blob/master/routers/juniper.php#L44) file as an example.